### PR TITLE
Anssi integration : add ANSSI profiles for NP targets

### PR DESCRIPTION
**Pull request by pthierry38**
Add French ANSSI 4 profiles for Not Protected target for Debian 8 distribution. profiles are not complete (i.e. lots of checks are missing) but the profile definition are here now.

### CODE REVIEW

File: Debian/8/input/profiles/anssi_np_nt28_average.xml [hunk 1]
```diff
@@ -0,0 +1,56 @@
+<Profile id="anssi_np_nt28_average">
+<title>Common Profile for ANSSI DAT-NT28 level average (fr: intermediaire) for Debian Systems</title>
```
Comment (redhatrises) [inline Debian/8/input/profiles/anssi_np_nt28_average.xml]: @pthierry38 please remove `Common`

File: Debian/8/input/profiles/anssi_np_nt28_high.xml [hunk 2]
```diff
@@ -0,0 +1,60 @@
+<Profile id="anssi_np_nt28_enforced">
```
Comment (redhatrises) [inline Debian/8/input/profiles/anssi_np_nt28_high.xml]: @pthierry38 this should be `anssi_np_nt28_high`

File: Debian/8/input/profiles/anssi_np_nt28_average.xml (hunk 1 above)
Comment (redhatrises) [inline Debian/8/input/profiles/anssi_np_nt28_average.xml]: @pthierry38 Wouldn't it make sense to say this for a title for consistency in this PR:
`Profile for ANSSI DAT-NT28 level average (intermediate)`

Does `for Debian systems` really need to be in the title?

File: Debian/8/input/profiles/anssi_np_nt28_minimal.xml [hunk 3]
```diff
@@ -0,0 +1,20 @@
+<Profile id="anssi_np_nt28_minimal">
+<title>Common Profile for ANSSI DAT-NT28 level minimal for Debian Systems</title>
```
Comment (redhatrises) [inline Debian/8/input/profiles/anssi_np_nt28_minimal.xml]: @pthierry38 can this be: `Profile for ANSSI DAT-NT28 Minimal Level` ?

File: Debian/8/input/profiles/anssi_np_nt28_high.xml [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+<title>Profile for ANSSI DAT-NT28 level high (enforced) for Debian Systems</title>
```
Comment (redhatrises) [inline Debian/8/input/profiles/anssi_np_nt28_high.xml]: @pthierry38 Can this be: `Profile for ANSSI DAT-NT28 High (Enforced) Level`?

File: Debian/8/input/profiles/anssi_np_nt28_restrictive.xml [hunk 5]
```diff
@@ -0,0 +1,60 @@
+<Profile id="anssi_np_nt28_restrictive">
+<title>Common Profile for General-Purpose Debian Systems</title>
```
Comment (redhatrises) [inline Debian/8/input/profiles/anssi_np_nt28_restrictive.xml]: @pthierry38 shouldn't this be `Profile for ANSSI DAT-NT28 Restrictive Level`?

**Comment by pthierry38** [comment]
Hello ! I've updated the content as you've specified and updated profiles using extends. Thanks for your comments.

**Comment by redhatrises** [comment]
Thanks @pthierry38. Made some typo corrections and pushed. Ack.

**Pull request merged**